doc/active_docs/account_management_api.json

```diff
@@ -7251,6 +7251,149 @@
         }
       }
     },
+    "/admin/api/personal/notification_preferences.json": {
+      "get": {
+        "summary": "Notification Preferences List",
+        "description": "Returns the list of notification preferences for the current user.",
+        "tags": [
+          "Notification Preferences"
+        ],
+        "parameters": [
+          {
+            "name": "access_token",
+            "in": "query",
+            "description": "A personal Access Token",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "success",
+            "content": {}
+          }
+        }
+      },
+      "put": {
+        "summary": "Notification Preferences Update",
+        "description": "Update the notification preferences for the current user.",
+        "tags": [
+          "Notification Preferences"
+        ],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/x-www-form-urlencoded": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "access_token": {
+                    "type": "string",
+                    "description": "A personal Access Token."
+                  },
+                  "application_created": {
+                    "type": "boolean",
+                    "description": "Application created."
+                  },
```

Comment:
This kind of definition allows me to click the checkbox "Send empty value":

![image](https://github.com/user-attachments/assets/d98cae51-be88-47fd-91a8-ad48525afaf3)

Which sends 'null' instead of `true` or `false`. IMO that's wrong, but the worst part is it actually stores `null` in the DB column:

![image](https://github.com/user-attachments/assets/d72e5e00-3b00-4c0b-af0d-cd1c9f7656f1)

I think `null` shouldn't be allowed, but if we want to allow it, then we should transform it to `false`.

Also, I think we should remove the "Send empty value" box from Swagger UI.